### PR TITLE
Heroku deployment

### PR DESCRIPTION
**Pull request by bfirsh**
https://embassynetwork-staging.herokuapp.com

Process of setting up an app is something like:

```
# update heroku CLI to have heroku.yml support
$ heroku update beta
$ heroku plugins:install @heroku-cli/plugin-manifest
# deploy app
$ heroku create your-app-name --manifest
$ heroku stack:set container
$ git push heroku master
```

Commits are well documented if you want to see my working.

~Based on top of #301, so that should be merged first.~ (Rebased!) Replaces #298.

### CODE REVIEW

File: modernomad/settings_docker.py [hunk 1]
```diff
@@ -58,3 +86,13 @@
...
+
+# Media storage
+AWS_STORAGE_BUCKET_NAME = env('AWS_STORAGE_BUCKET_NAME', default='')
+AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID', default='')
+AWS_SECRET_ACCESS_KEY = env('AWS_SECRET_ACCESS_KEY', default='')
+AWS_DEFAULT_ACL = 'public-read'
+if AWS_STORAGE_BUCKET_NAME:
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+    MEDIA_URL = env('MEDIA_URL', default='https://%s.s3.amazonaws.com/' % AWS_STORAGE_BUCKET_NAME)
+
```
Comment (jessykate) [inline modernomad/settings_docker.py]: Is AWS actually being used somewhere or is this just planning ahead?

File: modernomad/settings_docker.py [hunk 2]
```diff
@@ -41,7 +58,18 @@
 DISCOURSE_BASE_URL = env('DISCOURSE_BASE_URL', default='')
 DISCOURSE_SSO_SECRET = env('DISCOURSE_SSO_SECRET', default='')
```
Comment (jessykate) [inline modernomad/settings_docker.py]: We actually don't use discourse anymore so these two lines can be removed.

**Comment by jessykate** [comment]
I tried the "resolve conflicts" GUI option as the conflict was very simple, but for some reason the PR still showed up as un-resolvable. I manually did the rebase and have merged into master, so this is now complete.  YAY :dancer:

**Comment by bfirsh** [comment]
:tada: :tada: :tada:

File: modernomad/settings_docker.py (hunk 1 above)
Comment (bfirsh) [reply modernomad/settings_docker.py]: Yep, for file uploads. This is easily swappable out for other storage services if we choose - django-storages supports Google Cloud and so on.

**Pull request closed without merging**